packages/theme/css/foundations/size.css

```diff
@@ -31,7 +29,6 @@
   --salt-size-indicator: 3px;
   --salt-size-selectable: 16px;
   --salt-size-bar-strong: 12px;
-  /* --salt-size-bar-small: 4px; */
```

Comment:
this token was added/commented out in the `theme-updates` branch, removing it in this PR as no longer applicable